server/src/handlers/message_handler.rs

```diff
@@ -247,8 +258,23 @@ pub async fn get_all_topic_messages(
 ) -> Result<HttpResponse, actix_web::Error> {
     let topic_id: uuid::Uuid = messages_topic_id.into_inner();
 
-    let messages =
-        get_messages_for_topic_query(topic_id, dataset_org_plan_sub.dataset.id, &pool).await?;
+    let messages: Vec<models::Message> =
+        get_messages_for_topic_query(topic_id, dataset_org_plan_sub.dataset.id, &pool)
+            .await?
+            .into_iter()
+            .filter_map(|mut message| {
+                if message.content.starts_with("||[{") {
+                    match message.content.rsplit_once("}]") {
```

Comment:
In the format string }} = a single }. Not sure why they don't just do a \} but yeah.